RWKV-v5/src/model.py

```diff
@@ -1096,7 +1093,7 @@ def checkpointed_step(idx, targets, mask, last_shift_states,
             segment_size = self.ctx_len
 
             # Dummy 2D tensor of shape [B,0], are used to do "dummy checkpoint/forward/backprop" to keep everything in sync
-            dummy_empty_zero = torch.zeros(B,0, dtype=torch.long, device=cur_device)
+            dummy_empty_zero = torch.zeros(B, segment_size, dtype=torch.long, device=cur_device)
```

Comment:
Revert back to dummy empty zero - since the issue is isolated to lack of L2Wrap

Reply:
Causes issues with synchronisation.